docs/01-overview/main-areas/eventing/evnt-02-useful-links.md

```diff
@@ -6,9 +6,9 @@ Find out more about Eventing by following the links on this page.
 
 To learn more about how Eventing works, see:
 
-- [Eventing architecture](../../../05-technical-reference/03-architecture/evnt-01-architecture.md) - describes how Eventing works and the main actors involved, such as the Eventing Controller and Event Publisher Proxy. You can also find more information about event types here.
+- [Eventing architecture](../../../05-technical-reference/00-architecture/evnt-01-architecture.md) - describes how Eventing works and the main actors involved, such as the Eventing Controller and Event Publisher Proxy. You can also find more information about event types here.
 - [Event processing and delivery](../../../05-technical-reference/evnt-01-event-processing.md) - contains a diagram and explanation of the the Eventing processing and delivery flow.
-- [Subscription CRD](../../../05-technical-reference/06-custom-resources/evnt-01-subscription.md) - describes the `Subscription` CRD which you need to subscribe to events.
+- [Subscription CRD](../../../05-technical-reference/00-custom-resources/evnt-01-subscription.md) - describes the `Subscription` CRD which you need to subscribe to events.
```

Comment:
```suggestion
- [Subscription CRD](../../../05-technical-reference/00-custom-resources/evnt-01-subscription.md) - describes the `Subscription` CR, which you need to subscribe to events.
```
You don't subscribe a CRD, the D stands for Definition, you subscribe a CR, which is defined by a corresponding CRD.
This is probably for @alexandra-simeonova.